<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>back_business_day</class>
 <widget class="QDialog" name="back_business_day">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>800</width>
     <height>80</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>80</height>
    </size>
   </property>
   <property name="text">
    <string>   说明:日结是一个营业日的总结,即日结一次,营业日将跨过一天,如日结一次包括48小时甚至更多,
   那么所包含的时间段都归为一个营业日,系统提供修改最后一次日结的时间段的机会。如果中途
   出现日结错误,那么需要有限的人将日结日期返回,最后重新日结。</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>日结设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>上次日结时间:</string>
   </property>
  </widget>
  <widget class="lds_datetimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>381</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd hh:mm:ss</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>351</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>第一次日结可以修改上次日结时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>本次日结时间:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>411</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>本次日结时间由操作员设置,但不能大于当前时间</string>
   </property>
  </widget>
  <widget class="lds_datetimeEdit" name="dateTimeEdit_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>381</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd hh:mm:ss</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>营业日:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>营业日操作员可以选择,但修改最后一次日结,则不
能修改,所选营业日在上次与本次日结时间的有效
的有限日期。</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>440</y>
     <width>131</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>60</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="outer_stylesheet" stdset="0">
    <string>pushbutton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>131</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>60</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>反日结</string>
   </property>
   <property name="outer_stylesheet" stdset="0">
    <string>pushbutton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>131</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>60</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>最后日结</string>
   </property>
   <property name="outer_stylesheet" stdset="0">
    <string>pushbutton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>440</y>
     <width>131</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>60</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>日结</string>
   </property>
   <property name="outer_stylesheet" stdset="0">
    <string>pushbutton</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>381</width>
     <height>40</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_datetimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>lds_datetimeedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
